events/models.py

```diff
@@ -518,6 +535,20 @@ def get_enrolment_count(self):
         except AttributeError:
             return self.enrolments.count()
 
+    def get_attended_enrolment_count(self):
+        try:
+            # try to use an annotated value
+            return self.attended_enrolment_count
+        except AttributeError:
+            return self.enrolments.filter(attended=True).count()
```

Comment:
As the annotated value uses distinct, should distinct be added here as well?

Reply:
After discussion this should be fine, there shouldn't be need for distinct here because "enrolments" is the related_name in the ForeignKey references from Enrolment to Occurence and adding the boolean filter shouldn't change things either here AFAIK.